### PR TITLE
Maintenance: Simplify tarscripts

### PR DESCRIPTION
**Pull request by gardner48**
Simplify tarscript to include directories rather than individual files

### CODE REVIEW

File: scripts/kinsol
Comment (drreynolds) [inline scripts/kinsol:1]: Should this file also add the line
```
set -o pipefail
```

File: scripts/idas
Comment (drreynolds) [inline scripts/idas:1]: Should this file also add the line
```
set -o pipefail
```

File: scripts/ida
Comment (drreynolds) [inline scripts/ida:1]: Should this file also add the line
```
set -o pipefail
```

File: scripts/cvodes
Comment (drreynolds) [inline scripts/cvodes:1]: Should this file also add the line
```
set -o pipefail
```

File: scripts/cvode
Comment (drreynolds) [inline scripts/cvode:1]: Should this file also add the line
```
set -o pipefail
```

**Review by drreynolds — APPROVED** [review]

**Comment by balos1** [comment]
@gardner48 Is this still `dont-merge`?

**Review by balos1 — APPROVED** [review]

**Comment by gardner48** [comment]
> @gardner48 Is this still `dont-merge`?

Yes, there are few files/directories I need to exclude from the tarscript or remove from the repo

**Comment by gardner48** [comment]
> > @gardner48 Is this still `dont-merge`?
> 
> Yes, there are few files/directories I need to exclude from the tarscript or remove from the repo

Created two PRs (#366 and #367) to remove these files from the repo

**Comment by balos1** [comment]
@gardner48 now that #366 and #367 have been merged, can we remove the `dont-merge` tag on this PR?

**Comment by gardner48** [comment]
> @gardner48 now that #366 and #367 have been merged, can we remove the `dont-merge` tag on this PR?

Yes, should be good now

**Pull request merged**